### ISSUE TITLE
Export batch change list to CSV

### ISSUE THREAD

**Issue by chrispine** [sourcegraph/sourcegraph#39255]
([Slack discussion for context](https://sourcegraph.slack.com/archives/CMMTWQQ49/p1658428661544519))

Let's make it easy to export batch change data as CSV. We should discuss which columns it should include, but as a starting point for that discussion:

- name of (and/or link to) batch change; in cases where someone puts several BCs into the same spreadsheet
- link to PR
- status
- code-host?
- last activity (once we have this functionality; see Adam's Munich presentation, which I will not otherwise reference here)

**Comment by chrispine** [comment]
backlog?

**Comment by LawnGnome** [comment]
If we need a very rough estimate, probably 0.5-1d not including the last activity stuff?

**Comment by courier-new** [comment]
To me this one feels like it needs a good deal more information about how customers compile and maintain their manual spreadsheets today, too, before I would even attempt to estimate this. Just a couple questions that come to mind:
- What should/can we include for a changeset that the user downloading the CSV doesn't have access to?
- Do we include imported changesets?
- Does diff get exported at all? If so, in what form?

I'm sure there are many more.

**Comment by chrispine** [comment]
Yes, agreed. (I'd say we include all of the above and just add columns for "Imported", "HasAceess", etc; but always open to dissenting opinions.)

We recently pulled this out of the backlog, so I want to talk more about why before decided what the priority is here.
